### PR TITLE
Salesforce Marketing Cloud Connector User Guide v3.0.0

### PR DESCRIPTION
**Pull request by dhirsch1**

### CODE REVIEW

File: modules/ROOT/pages/salesforce/salesforce-mktg-connector.adoc [hunk 1]
```diff
@@ -1,250 +1,234 @@
...
 == Prerequisites
 
-This document assumes that you are familiar with Salesforce Marketing, Mule, Anypoint Connectors, Anypoint Studio, Mule concepts, elements in a Mule flow, and Global Elements.
+This document assumes that you are familiar with Salesforce Marketing Cloud, Mule, Anypoint Connectors, Anypoint Studio, Mule concepts, elements in a Mule flow, and global elements.
 
 You need login credentials to test your connection to your target resource.
 
 For hardware and software requirements and compatibility
-information, see the Connector Release Notes.
+information, see the connector Release Notes.
```
Comment (alexandruatomei9) [inline modules/ROOT/pages/salesforce/salesforce-mktg-connector.adoc]: This phrase should be on one line, I think.

Comment (dhirsch1) [reply modules/ROOT/pages/salesforce/salesforce-mktg-connector.adoc]: It looks OK in the preview view.

Comment (dhirsch1) [reply modules/ROOT/pages/salesforce/salesforce-mktg-connector.adoc]: It looks fine in the preview.

**Review by alexandruatomei9 — APPROVED** [review]

File: modules/ROOT/pages/salesforce/salesforce-mktg-connector.adoc (hunk 1 above)
Comment (alexandruatomei9) [reply modules/ROOT/pages/salesforce/salesforce-mktg-connector.adoc]: ok

**Review by alexandruatomei9 — APPROVED** [review]

File: modules/ROOT/pages/salesforce/salesforce-mktg-connector.adoc [hunk 2]
```diff
@@ -256,12 +240,12 @@ http://www.mulesoft.org/schema/mule/sfdc-marketing-cloud/current/mule-sfdc-marke
 	<http:listener-config name="HTTP_Listener_config" doc:name="HTTP Listener config">
 		<http:listener-connection host="localhost" port="8081" />
 	</http:listener-config>
-	<sfdc-marketing-cloud:sfdc-marketing-cloud-config name="Salesforce_Marketing_Cloud_Sfdc_marketing_cloud_config" doc:name="Salesforce Marketing Cloud Sfdc marketing cloud config">
-		<sfdc-marketing-cloud:basic-connection username="${config.username}" password="${config.password}" authEndPoint="${config.endpoint}" />
-	</sfdc-marketing-cloud:sfdc-marketing-cloud-config>
+	<sfdc-marketing-cloud:config name="Salesforce_Marketing_Cloud_Config" doc:name="Salesforce Marketing Cloud Config" doc:id="3e0ebd61-352b-4935-b18f-94ad8366233b" >
```
Comment (george-hoffer) [inline modules/ROOT/pages/salesforce/salesforce-mktg-connector.adoc]: Remove doc:id="" statements in code examples

File: modules/ROOT/pages/salesforce/salesforce-mktg-connector.adoc [hunk 3]
```diff
@@ -286,132 +270,106 @@ payload]]></ee:set-payload>
...
-The field Recurrence found in ScheduleDefinition, for example, is a complex field that has no structure.
+This map has a field called `minuteInterval` that belongs to a subclass of Recurrence called `MinutelyRecurrence`.
 
-To specify that we want to work with a MinutelyRecurrence, and not a Recurrence, we must manually
-add the fields belonging to the MinutelyRecurrence class, and add an
-extra field called concreteClassType of type String whose value is the name of the subclass.
+For the connector to use the MinutelyRecurrence object, you must also add the concreteClassType field with `MinutelyRecurrence` as the value.
 
-Here is how the mapping for the ScheduleDefinition would look in the Variables (vars) for our example:
+=== [[asynchronous-operations]]Asynchronous Operations
```
Comment (george-hoffer) [inline modules/ROOT/pages/salesforce/salesforce-mktg-connector.adoc]: Put the anchor before the heading, not on the same line.

File: modules/ROOT/pages/salesforce/salesforce-mktg-connector.adoc [hunk 4]
```diff
@@ -1,250 +1,234 @@
...
 
-. Create a new Mule Project by clicking on File > New > Mule Project.
-. Supply a name for your project and click Finish.
-+
-image::salesforce/salesforce-mktg-as-new-mule-project.png[New project dialog]
-+
-. Open pom.xml and under dependencies section add the following dependency for Mule Salesforce Marketing Connector, but first replace --VERSION-- with the latest version available:
+. Create a new Mule project by selecting *File > New > Mule Project*.
+. Supply a name for your project and click *Finish*.
+. Open the `pom.xml` file and add the following dependency for Mule Salesforce Marketing Connector. Mule translates the `RELEASE` dependency keyword to the current version.
```
Comment (george-hoffer) [inline modules/ROOT/pages/salesforce/salesforce-mktg-connector.adoc]: Change `pom.xml` to pom.xml

File: modules/ROOT/pages/salesforce/salesforce-mktg-connector.adoc [hunk 5]
```diff
@@ -1,250 +1,234 @@
...
 
 You need login credentials to test your connection to your target resource.
 
 For hardware and software requirements and compatibility
-information, see the Connector Release Notes.
+information, see the connector Release Notes.
 
-To use this connector with Maven, view the pom.xml dependency information in
-the Dependency Snippets in Anypoint Exchange.
+To use this connector with Maven, view the `pom.xml` dependency information in the Dependency Snippets section in Anypoint Exchange.
```
Comment (george-hoffer) [inline modules/ROOT/pages/salesforce/salesforce-mktg-connector.adoc]: Remove back quotes from pom.xml

File: modules/ROOT/pages/salesforce/salesforce-mktg-connector.adoc [hunk 6]
```diff
@@ -1,250 +1,234 @@
...
+Anypoint Connector for Salesforce Marketing Cloud (Salesforce Marketing Cloud Connector) lets you connect to the Marketing Cloud API, which is also called Salesforce Marketing Cloud. This connector exposes convenient operations for using the capabilities of Salesforce Marketing Cloud.
 
 == Prerequisites
 
-This document assumes that you are familiar with Salesforce Marketing, Mule, Anypoint Connectors, Anypoint Studio, Mule concepts, elements in a Mule flow, and Global Elements.
+This document assumes that you are familiar with Salesforce Marketing Cloud, Mule, Anypoint Connectors, Anypoint Studio, Mule concepts, elements in a Mule flow, and global elements.
 
 You need login credentials to test your connection to your target resource.
 
 For hardware and software requirements and compatibility
```
Comment (george-hoffer) [inline modules/ROOT/pages/salesforce/salesforce-mktg-connector.adoc]: There's no hardware requirements in the release notes -- remove "hardware and"

**Review by george-hoffer — APPROVED** [review]

**Pull request merged**